pkg/server/etcd/watch.go

```diff
@@ -161,6 +161,8 @@ func (w *watcher) Cancel(id int64, err error, compact bool) {
 	var tags []metrics.T
 	if compact {
 		tags = append(tags, metrics.Tag("compact", "true"))
+	} else {
```

Comment:
  Would it be better to fix the issue in this way ？
```
tag = append(tags, metrics.Tag("compact",  strconv.FormatBool(compact)
```

Reply:
OK